sycl/include/CL/sycl/backend.hpp

```diff
@@ -275,4 +279,4 @@ make_kernel_bundle(const typename backend_traits<Backend>::template input_type<
   return detail::createSyclObjFromImpl<kernel_bundle<State>>(KBImpl);
 }
 } // namespace sycl
-} // __SYCL_INLINE_NAMESPACE(cl)
+} // __SYCL_INLINE_NAMESPACE(cl)
```

Comment:
Did clang-format remove the new-line at the end of the files?

Reply:
No, I've removed them manually after comparing with an external app

Reply:
I was worried there for a moment. We normally require a newline at the end of files, so please add them back. 😄